ios/FluentUI/Vnext/Notification/FluentNotification.swift

```diff
@@ -32,50 +32,34 @@ import SwiftUI
     /// Action to be dispatched by tapping on the toast/bar notification.
     var messageButtonAction: (() -> Void)? { get set }
 
-    /// Action to be dispatched when dismissing toast/bar notification.
-    var dismissAction: (() -> Void)? { get set }
-
     /// Design token set for this control, to use in place of the control's default Fluent tokens.
     var overrideTokens: NotificationTokens? { get set }
 }
 
 /// View that represents the Notification.
-public struct NotificationViewSwiftUI: View, ConfigurableTokenizedControl {
-    @Environment(\.horizontalSizeClass) var horizontalSizeClass: UserInterfaceSizeClass?
-    @Environment(\.fluentTheme) var fluentTheme: FluentTheme
-    @ObservedObject var state: MSFNotificationStateImpl
-    @Binding var isPresented: Bool
-    @State private var bottomOffsetForDismissedState: CGFloat = 0
-    @State private var bottomOffset: CGFloat = 0
-    let defaultTokens: NotificationTokens = .init()
-    var tokens: NotificationTokens {
-        let tokens = resolvedTokens
-        tokens.style = state.style
-        return tokens
-    }
-
-    public func overrideTokens(_ tokens: NotificationTokens?) -> NotificationViewSwiftUI {
+public struct FluentNotification: View, ConfigurableTokenizedControl {
+    public func overrideTokens(_ tokens: NotificationTokens?) -> FluentNotification {
         state.overrideTokens = tokens
         return self
     }
 
     public init(style: MSFNotificationStyle,
```

Comment:
Do you mind adding documentation comments for the public initializer? Documenting the params here will be important for the client code.